buyer-api/src/routes/dataOrders/getDataOrder.js

```diff
@@ -31,6 +31,12 @@ router.get(
  * /orders/{id}:
  *   get:
```

Comment:
We should document the structure of the responded resource, I'm adding this to a private improvements document.